dhis2/src/main/java/org/motechproject/dhis2/service/impl/SettingsServiceImpl.java

```diff
@@ -60,6 +60,10 @@ private synchronized void loadSettings() {
             LOGGER.debug("Loading DHIS2 settings...");
             Gson gson = new Gson();
             settings = gson.fromJson(jsonText, Settings.class);
+            String URI = settings.getServerURI();
+            if(settings.getServerURI().endsWith("/")) {
```

Comment:
why using settings.getServerURI() if you aleeady have that in a variable? (1 line above this)